box-share-sdk/src/main/java/com/box/androidsdk/share/views/CollaboratorsInitialsView.java

```diff
@@ -158,44 +159,50 @@ public void run() {
             };
 
 
-    public void updateUi(BoxIteratorCollaborations boxIteratorCollaborations) {
+    private void updateView(BoxIteratorCollaborations boxIteratorCollaborations) {
         mCollaborations = boxIteratorCollaborations;
-        if (mCollaborations != null && mCollaborations.size() != 0) {
+        if (mCollaborations != null) {
```

Comment:
In case this mCollaborations is null, shouldn't we setVisibility to "GONE" ?
I feel like you could check:
if(mCollaborations == null || mCollaborations.size() == 0)  --> view.GONE
otherwise the view gets VISIBLE and you can continue processing